src/frontend/cypress/e2e/login-page/loginpage.cy.ts

```diff
@@ -53,8 +53,8 @@ describe('Test DPIA Login Page', () => {
 
     const password = Cypress.env('PASSWORD');
 
-    cy.get('.d-none').click();
-    cy.get('#user').type('DORGALE');
+    cy.get('.d-none',{ timeout: 10000 }).click();
+    cy.get('#user',{ timeout: 10000 }).type('DORGALE');
```

Comment:
Wondering if we will define `{timeout: 10000}` for every field we write in the future? 🤔 Not so elegant I guess, how about  `defaultCommandTimeout: 10000,`?
ref: https://stackoverflow.com/a/59205575

Can you please have a look if this fits into our usecase?

Reply:
Thank you so much @kushal-arora-fw for the suggestion. The Timeout is added globally now!